index.md

```diff
@@ -162,3 +162,10 @@ Rather than being tied to a single language or backing implementation,
 **pmemkv** provides different options for language bindings and storage engines.
 
 See the [pmemkv page](../pmemkv/) for available documentation.
+
+#### libpmemset
```

Comment:
since this is part of pmdk repo it should be above in this file, along with other pmdk's libraries... I believe

Reply:
I have moved it below librpmem.